use std::process::ExitCode;

fn main() -> ExitCode {
    return ExitCode::FAILURE;
}
